applications/portfolioDrafts/package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "description": "ATAT Web API Portfolio Draft support",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint --ext .js,.ts,.json -c ../../.eslintrc.json ."
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "lint": "eslint --ext .js,.ts,.json -c ../../.eslintrc.json .",
+    "lint:fix": "npm run lint -- --fix"
   },
   "dependencies": {
     "@aws-sdk/client-dynamodb": "^3.23.0",
     "@aws-sdk/lib-dynamodb": "^3.23.0",
+    "@aws-sdk/util-dynamodb": "^3.23.0",
```

Comment:
Do we need `@aws-sdk/util-dynamodb` for something ourselves? Is this a peer dependency? I have no problem with adding it; I just want to make sure I understand for future applications

Reply:
It's a peer dependency

Reply:
I'll make a commit to remove it, its a peer dependency of `"node_modules/@aws-sdk/lib-dynamodb"`